File: Data Structures & Algorithms/binary-search/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number}
     */
    search(nums, target) {
        let res = -1;
        for(let n = 0; n < nums.length; n++){
         if(nums[n] == target){
            res = n;
         }
        }
        if(res == -1)return -1;
        return res;
    }
}

File: Data Structures & Algorithms/subsets/submission-1.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        length = len(nums)
        maxsub = 2 ** length
        res = []
        for i in range(maxsub):
            cursub = []
            for j in range(length):
                if i & (1 << j):
                    cursub.append(nums[j])
            res.append(cursub)
        return(res)

        